### PR TITLE
Correction: variable's value must be an integer

### LINKED ISSUE #3089: set variable value to text string?
I'd like to set a variable value to a text string. The _Reference Manual_ appears to contain conflicting info about whether this ought to be possible. Clarification about the intended behavior would be much appreciated!

==Details==
The manual page for [to.set_variable](https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to/set-variable/) clearly indicate strings are permitted. However, the page for [variable_if, variable_unless](https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/conditions/variable/) seems to indicate otherwise. See attached screenshots below.

In practice, Karabiner simply treats the following rule as non-existent (i.e. pressing the "x" key returns the letter "x" and does not set the variable to the value). 

```
      "rules" : [ {
        "description" : "variable to string",
        "manipulators" : [ {
          "from" : {
            "key_code" : "x"
          },
          "to" : [ {
            "set_variable" : {
              "name" : "i_am_text",
              "value" : "string"
            }
          } ],
          "type" : "basic"
        } ]
      } ]
```

**Screenshot 1** ⮕ value shown as string
---
image

**Screenshot 2** ⮕ value NOT shown as string 
---
image

### LINKED ISSUE #3091: Bug? `set_variable` only works with integer as value
It took me a while to figure this out, cause I could not make `set_variable` work, until a looked at other modifications and noticed they were all using integers as values. The docs, however, not only do not indicate this requirement, but even suggest a string in the example, which is in fact not working.

So either the docs are erroneous, or there is a bug that I cannot set variables to a non-integer value. (I tried string & boolean.)

### PR DESCRIPTION
**Pull request by MuhammedZakir**
Resolves: pqrs-org/Karabiner-Elements#3089
Resolves: pqrs-org/Karabiner-Elements#3091

### CODE REVIEW

File: sites/karabiner-elements/content/en/docs/json/complex-modifications-manipulator-definition/conditions/variable/index.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ Change an event if/unless the variable is the specified value.
...
-| ------------- | ------------ | --------------------------------------- |
-| `type`        | **Required** | `"variable_if"` or `"variable_unless"`. |
-| `name`        | **Required** | Target variable name.                   |
-| `value`       | **Required** | Target variable value.                  |
-| `description` | Optional     | A human-readable comment                |
+| Name          | Required     | Description                                 |
+| ------------- | ------------ | ------------------------------------------- |
+| `type`        | **Required** | `"variable_if"` or `"variable_unless"`.     |
+| `name`        | **Required** | Target variable name.                       |
+| `value`       | **Required** | Target variable value (must be an integer). |
```
Comment (MuhammedZakir) [inline sites/karabiner-elements/content/en/docs/json/complex-modifications-manipulator-definition/conditions/variable/index.md:26]: Shall I change the description to `An integer, string or boolean.`?

**Comment by tekezo** [comment]
Thanks!

File: sites/karabiner-elements/content/en/docs/json/complex-modifications-manipulator-definition/conditions/variable/index.md (hunk 1 above)
Comment (MuhammedZakir) [reply sites/karabiner-elements/content/en/docs/json/complex-modifications-manipulator-definition/conditions/variable/index.md:26]: @tekezo: Did you accidentally merge this? This is not accurate anymore as it now accepts both string and boolean. We need do what I said above. :-)

Comment (tekezo) [reply sites/karabiner-elements/content/en/docs/json/complex-modifications-manipulator-definition/conditions/variable/index.md:26]: Thanks! I updated 👍 
https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to/set-variable/

**Comment by MuhammedZakir** [comment]
You're welcome and thanks for your work! :-D

**Pull request merged**